packages/v4/src/content/accessibility/code-editor/code-editor.md

```diff
@@ -0,0 +1,44 @@
+---
+id: Code editor
+section: components
+---
+
+A **code editor** is the open sourced monaco code editor wrapped in PatternFly controls and styles. PatternFly does not 
+manage the interactions or the code for the monaco editor itself. 
+
+**Keyboard users** should be able to focus on the code editor controls using **Tab** to move forward and 
+**Tab + Shift** to move backward through interactive elements. They should be able to select a focused control, 
+such using **Space** or **Enter**. Keyboard users should also be able to move focus onto the monaco
+code editor using **Tab**, and then into the monaco code editor using the **Down arrow**. If focus is in the monaco 
```

Comment:
Nit: rather than "They should be able to select a focused control, such using....", maybe "They should be able to select a focused control using..."